app/models/order.rb

```diff
@@ -0,0 +1,16 @@
+class Order < ApplicationRecord
+  belongs_to :client
+  belongs_to :staff
+  has_many :line_items
+  validates :total, presence: true
+  validates :subtotal, presence: true
+  validates :tax, presence: true
+
+  # TODO: write code to grab line_item_total for each line_item to get subtotal for order
+  def calculate_subtotal(line_items)
+  end
+
+  def calculate_tax
+    calculate_subtotal * 0.00575
```

Comment:
Ya, I wasn't completely sure what to do with tax. Russell said not to make connect it to some other website or api so maybe we can make a way for the admin to set the tax on an order or something.